app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/insurancedetail/ContractDetailDestination.kt

```diff
@@ -151,11 +173,6 @@ private fun ContractDetailScreen(
               .only(WindowInsetsSides.Horizontal + WindowInsetsSides.Bottom)
               .asPaddingValues()
               .plus(PaddingValues(top = 16.dp)),
-            modifier = Modifier
-              .fillMaxSize()
-              .consumeWindowInsets(
-                WindowInsets.safeDrawing.only(WindowInsetsSides.Horizontal + WindowInsetsSides.Bottom),
-              ),
```

Comment:
Did these break something?
They were here to consume those insets since the insets are applied to the lazyColumn using `asPaddingValues()`, but that function does not consume the insets for the children, it only uses them and leaves them untouched, so we had to do this manually here.
It may need to stay here for this reason.
Now that we are consuming them higher up, I am not sure if this modifier is actually affected 👀
Because I know that it does not set the consumer modifier local, but I am not sure if it at least reads from it only.